Collector/Twig/SlugifyExtension.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Http\HttplugBundle\Collector\Twig;
+
+/**
+ * @author Ibrahima SOW <sowbiba@hotmail.com>
+ */
+class SlugifyExtension extends \Twig_Extension
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @return array
+     */
+    public function getFilters()
+    {
+        return [
+            new \Twig_SimpleFilter('slugify', [$this, 'slugify'], ['is_safe' => ['html']]),
```

Comment:
i guess there are quite some twig filters out there that provide a function with the same name.

i agree to not introduce a dependency on some twig thingy, and the actual slugify looks simple enough that we don't need a library for that. but i think we need a more specific name for this to avoid collisions. maybe `httplug_slugify` ?